### PR TITLE
Update installation instructions

### PR DESCRIPTION
**Pull request by pvdb**
@sstephenson / @mislav - Just a minor update to the installation instructions for `rbenv/rbenv-vars`, to bring them up-to-date, and in line with the installation instructions for `rbenv/rbenv-default-gems`... thanks!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,13 +6,9 @@ before spawning Ruby processes.
 
 ## Installation
 
-To install rbenv-vars, clone this repository into your
-`~/.rbenv/plugins` directory. (You'll need a recent version of rbenv
-that supports plugin bundles.)
+Make sure you have the latest rbenv and ruby-build versions, then run:
```
Comment (mislav) [inline README.md]: Can you remove the mention of ruby-build? It's not needed to use rbenv-vars.

Comment (pvdb) [inline README.md]: Good catch... the copy-n-paste :imp: strikes again!  Removed in 385750f9943aea304e40dd97a38456c70655619a

**Comment by pvdb** [comment]
This should be good to go now, @mislav ... thanks!

**Comment by pvdb** [comment]
Poke @mislav :smiley:

File: README.md [hunk 2]
```diff
@@ -6,13 +6,9 @@ before spawning Ruby processes.
...
 
-To install rbenv-vars, clone this repository into your
-`~/.rbenv/plugins` directory. (You'll need a recent version of rbenv
-that supports plugin bundles.)
+Make sure you have the latest version of rbenv, then run:
 
-    $ mkdir -p ~/.rbenv/plugins
-    $ cd ~/.rbenv/plugins
-    $ git clone https://github.com/rbenv/rbenv-vars.git
+    git clone https://github.com/rbenv/rbenv-vars.git $(rbenv root)/plugins/rbenv-vars
```
Comment (mislav) [inline README.md]: I just realized that `plugins/` directory is not guaranteed to exist at first. So we still need the line:

```
mkdir -p `$(rbenv root)/plugins`

Comment (pvdb) [inline README.md]: Thanks @mislav but `git clone` will take care of creating what it calls the "leading directories", so there's no need to explicitly create the `plugins/` directory beforehand:

```shell
$ pwd
/tmp/blegga
$  tree .
.

0 directories, 0 files
$ git clone https://github.com/rbenv/rbenv-vars.git ${PWD}/foo/bar/qux/plugins/rbenv-vars
Cloning into '/tmp/blegga/foo/bar/qux/plugins/rbenv-vars'...
remote: Counting objects: 90, done.
remote: Total 90 (delta 0), reused 0 (delta 0), pack-reused 90
Unpacking objects: 100% (90/90), done.
Checking connectivity... done.
Running 'git submodule update' from post-checkout hook...
$ tree .
.
└── foo
    └── bar
        └── qux
            └── plugins
                └── rbenv-vars
                    ├── LICENSE
                    ├── README.md
                    ├── bin
                    │   └── rbenv-vars
                    └── etc
                        └── rbenv.d
                            └── exec
                                └── rbenv-vars.bash

9 directories, 4 files
$ _
```

Comment (mislav) [inline README.md]: > git clone will take care of creating what it calls the "leading directories"

Wooow mind = blown, thanks!

**Pull request merged**